mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -47,6 +47,7 @@ private InputFormatConfig() {
   public static final String TABLE_SCHEMA = "iceberg.mr.table.schema";
   public static final String PARTITION_SPEC = "iceberg.mr.table.partition.spec";
   public static final String SERIALIZED_TABLE_PREFIX = "iceberg.mr.serialized.table.";
+  public static final String TABLE_CATALOG_PREFIX = "iceberg.mr.table.catalog";
```

Comment:
nit: could we add "." at the end of the prefix? I think it looks a little better in the config.

Reply:
Done